src/main/java/org/primefaces/util/MapBuilder.java

```diff
@@ -56,4 +57,8 @@ private MapBuilder(Map<K, V> map) {
     public Map<K, V> build() {
         return map;
     }
+
+    public Map<K, V> ibuild() {
+        return Collections.unmodifiableMap(map);
+    }
```

Comment:
What call it iBuild?  What does the "i" stand for?  Maybe Javadoc this method too?

Reply:
ahah I expected that 😄 It means "immutable"